library/Notifications/Widget/ItemList/IncidentListItem.php

```diff
@@ -73,19 +80,22 @@ protected function assembleTitle(BaseHtmlElement $title): void
     protected function assembleHeader(BaseHtmlElement $header): void
     {
         $header->add($this->createTitle());
+        $meta = new HtmlElement('span', Attributes::create(['class' => 'meta']));
+
+        /** @var Source $source */
+        $source = $this->item->object->source;
+        $meta->addHtml((new SourceIcon(SourceIcon::SIZE_BIG))->addHtml($source->getIcon()));
 
         if ($this->item->recovered_at !== null) {
-            $header->add(Html::tag(
-                'span',
-                ['class' => 'meta'],
-                [
-                    'closed ',
-                    new TimeAgo($this->item->recovered_at->getTimestamp())
-                ]
-            ));
+            $meta->add([
+                $this->translate('closed') . ' ',
+                new TimeAgo($this->item->recovered_at->getTimestamp())
+            ]);
```

Comment:
With this, a translator has the option to move the time representation.

```suggestion
           $meta->addHtml(FormattedString::create(
               $this->translate('closed %s', '(incident) ... <relative time>'),
               new TimeAgo($this->item->recovered_at->getTimestamp())
           ));
```